src/pages/execution/ExpertSystemRules.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactNode } from "react";
+import { ExecutionAndTasks } from "../../types/execution";
+import { timestampRenderer } from "../../utils/helpers";
+import { Alert } from "@mui/material";
+
+export type Severity = "ERROR" | "WARNING" | "INFO";
+
+export type AlertItem = {
+  component: ReactNode;
+  severity: Severity;
+};
+
+export type Rule = (
+  executionAndTasks: ExecutionAndTasks,
+  severity: Severity,
```

Comment:
Rules don't depend on the max severity so we can leave it out of the input parameters.